app/src/main/java/com/p2p/presentation/truco/actions/TrucoActionsBottomSheetFragment.kt

```diff
@@ -105,18 +148,36 @@ class TrucoActionsBottomSheetFragment : BaseBottomSheetDialogFragment<ViewTrucoA
 
     private fun onCollapsed() {
         binding.openButtonIcon.animateRotation(0f)
-        changeVisibleButtonsEnable(false)
+        disableEnvidoAndTrucoButtons()
```

Comment:
Y esto mantenerlo como estaba
```suggestion
        changeVisibleButtonsEnable(false, false)
```